charts/unifi/values.yaml

```diff
@@ -169,6 +169,27 @@ discoveryService:
   ## Set the externalTrafficPolicy in the Service to either Cluster or Local
   # externalTrafficPolicy: Cluster
 
+syslogService:
+  type: NodePort
```

Comment:
You used NodePort here because it needs to be reachable by UniFi devices on the network and thus ClusterIP wouldn't be a good choice, correct?

Reply:
Yes, it should be configured the same way as `controllerService` or `stunService` and also respond to the same IP (ie. what devices consider the "controller address") to make it work, I think.

Reply:
FWIW, I configured a `LoadBalancer` for all of the services which has the IP `192.168.30.158`, and then added a DNS entry `unifi` so that access points would automatically connect to that IP and register with the controller (what Unifi calls [Layer 3 Adoption](https://help.ui.com/hc/en-us/articles/204909754-UniFi-Layer-3-Adoption-for-Remote-UniFi-Controllers#6)).

After enabling the "remote syslog" option, access points got this configuration:

```bash
BedroomAP-BZ.v4.3.21# cat /tmp/system.cfg | grep syslog
# syslog
syslog.level=7
syslog.remote.encrypt=enabled
syslog.remote.ip=192.168.30.158
syslog.remote.key=<redacted>
syslog.remote.port=5514
syslog.remote.status=enabled
syslog.status=enabled
```
My guess is that the controller just configures the `syslog.remote.ip` with the IP that was determined as the "controller IP" during adoption, in my case because of the DNS entry - but there could be some other magic at play. Unfortunately, I couldn't find any official documentation on the Unifi site for this feature.